taggit/models.py

```diff
@@ -68,8 +68,11 @@ def save(self, *args, **kwargs):
             except IntegrityError:
                 pass
             # Now try to find existing slugs with similar names
-            slugs = set(Tag.objects.filter(slug__startswith=self.slug)
-                                   .values_list('slug', flat=True))
+            slugs = set(
+                self._default_manager
```

Comment:
If I may suggest `_default_manager` should be accessed with the class not the instance as demonstrated in the majority of cases in Django's code base. If I understand correctly it has something to do how the default manager adds itself to the model class during import time and not the instance on creation time which in theory could be prevented if a Tag subclass were to fiddle with its class attributes during class or instance creation.